.travis.yml

```diff
@@ -8,7 +8,9 @@ rvm:
 
 install:
   - bundle install
+  - yarn global add raml-cop
```

Comment:
This might be a really dumb question but can we add raml-cop using yarn? I have seen it to be part of npm?

Reply:
never mind, just saw your screenshot

Reply:
Any `npm` package can be installed with `yarn`. `yarn` is really just a drop in replacement for the `npm` CLI - they talk to the same registry through different proxies.